operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -3,24 +3,28 @@
 **NOTE:** Some of the documentation linked from this page mentions use of the Boot Orchestration Service (BOS). The use of BOS
 is only relevant for booting compute nodes and can be ignored when working with NCN images.
 
-This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image.
+This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image
+with minor command modifications.
 
-1. Identify the NCN image to be modified.
+1. (`ncn-mw#`) Identify the NCN image to be modified.
 
-    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by Kubernetes NCNs.
-    However, the steps are the same for any Management NCN SquashFS image.
+    If this procedure is being done as part of a CSM upgrade, then the documentation which linked to this procedure will have
+    provided instructions for how to set the `ARTIFACT_VERSION` variable.
 
-    If the image to be modified is the image currently booted on a Kubernetes NCN, the value for `ARTIFACT_VERSION` can be found by looking
-    at the boot parameters for the NCNs, or from `/proc/cmdline` on a booted Kubernetes NCN. The version has the form of `X.Y.Z`.
-    See: [boot parameters](../../background#metalserver)
+    Otherwise, if the image to be modified is the image currently booted on an NCN, then find the value for `ARTIFACT_VERSION`
+    by looking at the boot parameters for the NCNs, or by reading  `/proc/cmdline` on a booted NCN. The version has the form of `X.Y.Z`.
+    See: [boot parameters](../../background#metalserver).
```

Comment:
If it's the first time modifying an image, everything here is accurate.  However, if you were to come back to this procedure _after_ doing the initial customization, this would no longer be accurate.  At that point, the artifact would not have X.Y.Z in the name.  It would be an IMS-generated UUID.  See lines 165-166.

Reply:
Yeah, I noticed that. Since this PR was specifically to help with the upgrade case, I didn't want to get bogged down fixing that issue as well (since in the upgrade case, they will be customizing the image that was just uploaded into S3 in stage 0).

I'll open another ticket to make the further corrections to this. My PR just retains the existing language for this part of it, so it isn't introducing any new problems in that regard, just making the one case clearer.

Reply:
I've opened [CASMINST-5588](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5588) for this work. It will also cover [Eli's comment](https://github.com/Cray-HPE/docs-csm/pull/2790/files#r1014380369) (other than the broken link he pointed out, which I will fix here).